crates/core/src/vfs.rs

```diff
@@ -488,26 +479,15 @@ impl OpenFile {
     ///
     /// * Panics if the `Node` has no content
     pub fn from_node<P, S: IndexedFull>(repo: &Repository<P, S>, node: &Node) -> Self {
-        let mut start = 0;
-        let mut content: Vec<_> = node
-            .content
-            .as_ref()
-            .unwrap()
-            .iter()
-            .map(|id| {
-                let starts_at = start;
-                start += repo.index().get_data(id).unwrap().data_length() as usize;
-                BlobInfo { id: *id, starts_at }
-            })
-            .collect();
+        let content: Vec<_> = node.content.as_ref().unwrap().clone();
```

Comment:
Can we replace the `unwrap` with an `expect` here, as we aim to reduce the number of unwraps (without any further information), I thought. We should probably return a `RusticResult<Self>` here though and use an error, because `rustic_core` shouldn't panic as a library as that's harder to catch on the consumer side.

Reply:
It's fine to make this api breaking, as we have a lot of other api breaking code already coming up.

Reply:
here we can actually just use an empty vec.